ngraph/python/src/ngraph/opset1/__init__.py

```diff
@@ -0,0 +1,134 @@
+# ******************************************************************************
+# Copyright 2017-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+"""ngraph module namespace, exposing factory functions for all ops and other classes."""
+# noqa: F401
+
+from pkg_resources import get_distribution, DistributionNotFound
+
+try:
+    __version__ = get_distribution("ngraph-core").version
+except DistributionNotFound:
+    __version__ = "0.0.0.dev0"
```

Comment:
Please remove this code, it should only be in `ngraph/__init__.py`. It specifies the version of nGraph used.

Reply:
Done